polkadot-parachains/src/chain_spec.rs

```diff
@@ -188,10 +188,13 @@ pub type StatemineChainSpec =
 	sc_service::GenericChainSpec<statemine_runtime::GenesisConfig, Extensions>;
 pub type WestmintChainSpec =
 	sc_service::GenericChainSpec<westmint_runtime::GenesisConfig, Extensions>;
+pub type RockmineChainSpec =
+	sc_service::GenericChainSpec<rockmine_runtime::GenesisConfig, Extensions>;
```

Comment:
including another runtime will increase compile times quite a bit, we will want to look into adding conditional compilation here, similar to Polkadot https://github.com/paritytech/polkadot/blob/3bc7afae4145a06fac75e772d44a46ab824d3a93/cli/src/command.rs#L96